napari/_qt/menus/file_menu.py

```diff
@@ -160,7 +162,31 @@ def _rebuild_samples_menu(self):
 
         self.open_sample_menu.clear()
 
-        for plugin_name, samples in plugin_manager._sample_data.items():
+        # eg ('dock', ('my_plugin', {'My widget': MyWidget}))
+        _iterable: Iterator[Tuple[str, Dict[str, SampleDict]]]
+        try:
+            import npe2
+        except ImportError:
+            _iterable = iter([])
+        else:
+            pm = npe2.PluginManager.instance()
+            _iterable = (
+                (
+                    plugin_name,
+                    {
+                        c.key: {
+                            'data': c.open,
+                            'display_name': c.display_name,
```

Comment:
@tlambert03 could we wrap the display name with a `trans._(dname)` call?

We have not updated the translation parsing for the manifest files (yet) but this is a good reminder :)

Reply:
help me understand the flow here.  Since this `display_name` is coming from the manifest, would that suggest that we need to use `trans._` _everywhere_ we pull text from the manifest?  I would have assumed we could do that conversion somewhere higher up in the chain, rather than at the point of usage of the string

Reply:
>conversion somewhere higher up in the chain, rather than at the point of usage of the string

Well yes, we could do that wherever we parse the manifest then. We just need to to it somehwere :)

Reply:
yeah, lets do that :)

Reply:
Sounds good!